common/src/main/resources/tachyon-default.properties

```diff
@@ -94,20 +92,20 @@ tachyon.worker.network.netty.file.transfer=MAPPED
 tachyon.worker.memory.size=128MB
 tachyon.worker.network.netty.watermark.high=32KB
 tachyon.worker.network.netty.watermark.low=8KB
-tachyon.worker.allocate.strategy.class=tachyon.worker.block.allocator.MaxFreeAllocator
```

Comment:
try to shorten the property name a bit---if people don't like it, I could revert these back